app/assets/javascripts/initializers/initializeOnboardingTaskCard.js.erb

```diff
@@ -8,7 +8,13 @@ function initializeOnboardingTaskCard() {
 
   var taskCard = document.getElementsByClassName("onboarding-task-card")[0];
 
-  if (taskCard) {
+  if (taskCard == null) { return }
```

Comment:
```suggestion
  if (taskCard === null) { return }
```

Reply:
I should've left a note that I did this on purpose. This actually catches both [undefined and null](https://stackoverflow.com/a/2647888/5494108).